user/languages/android.md

```diff
@@ -204,6 +204,8 @@ For Android projects, `env` and `jdk` can be given as arrays to construct a buil
 
 The `dist: trusty` build environment is the only supported build environment for Android but if you would like to build on newer build environments e.g. `dist: jammy`, you can exercise your access to the Travis CI build environments and install required packages and tools. An example .travis.yml config can be reviewed below:
 
+> Sleep values may need adjusting depending on the specific build environment
```

Comment:
@qasim-at-tci  here it's out of context. What I meant was an inline .travis.yml comment inline to `  - sleep 15`

Reply:
@michal-at-travisci Updated inline.